Changes

```diff
@@ -72,6 +72,9 @@ _______________
   command-line of the toplevel.
   (Nicolás Ojeda Bär, review by Florian Angeletti, report by Daniel Bünzli)
 
+- #12919: Fix register corruption in caml_callback2_asm on s390x.
+  (Miod Vallat, review by Gabriel Scherer)
```

Comment:
@dustanddreams sorry for the tedious stuff, but can you move this Changes entry down blow in the 5.2 section?

Reply:
Done.